### ISSUE TITLE
Chubgraph must be declared before every instance of it in the code

### ISSUE THREAD

**Issue by mariobuoninfante** [ccrma/chuck#141]
Ubuntu 16.04 LTS 64bit - ChucK 1.4.0.0

I noticed that it is not possible to use a Chubgraph if its declaration comes after its instance
When running the following code I get this error: _[untitled]:line(1): cannot resolve operator '=>' on types 'C' and 'DAC'..._

```
C My_C => dac;

private class C extends Chubgraph
{
    SinOsc S => outlet;
}

while(true)
    second => now;
```

**Comment by armand-bernardi** [comment]
I confirm this issue.

**Comment by heuermh** [comment]
See also #107

**Comment by gewang** [comment]
this has been addressed (hopefully!) and will be part of chuck-1.5.2.0
